src/scss/settings/_variables.scss

```diff
@@ -125,7 +125,7 @@ $border-radius: 4px;
 
 $zIndex: (
     lowest  : -1,
-    low     : 10,
+    low     : 1,
```

Comment:
Does anything else use `low` atm?

If so, might be worth adding a new key value for `1`.  If this isn't an issue though ignore me though :)

Reply:
Nothing within Fozzie uses it right now. Are these values exposed out?

Reply:
Yeh – they can be used by other modules/projects.  I highly doubt it is though (or that changing it to `1` will have any impact).

If it's ok within Fozzie happy for this to go in as is.